x-pack/solutions/security/plugins/security_solution/server/lib/entity_analytics/privilege_monitoring/constants.ts

```diff
@@ -13,9 +13,11 @@ export const TIMEOUT = '10m';
 export const INTERVAL = '10m';
 
 export const PRIVILEGE_MONITORING_ENGINE_STATUS = {
-  INSTALLING: 'installing',
+  // TODO Make the engine initialization async before uncommenting these lines
+  // Also implement a status API for FE to poll
+  // INSTALLING: 'installing',
```

Comment:
@tiansivive I updated the init API to return `STARTED` instead of `INSTALLING`, and commented out `STOPPED` and `INSTALLING ` statuses because they are no longer used.
If we make the `init` API async, we have to create an API for polling the status so we can update the UI accordingly.